Dockerfile

```diff
@@ -21,7 +21,13 @@ RUN pip install \
 # napalm is used for gathering information from network devices
       napalm \
 # ruamel is used in startup_scripts
-      ruamel.yaml
+      ruamel.yaml \
+# pinning django to the version required by netbox
+# adding it here, to install the correct version of
+# django-rq
+      'Django>=1.11,<2.1' \
```

Comment:
You could make this dynamic with a default.

Reply:
I don't get what you mean, could you elaborate?

Reply:
Sorry, my eyes were already too tired.. They just saw the one version number 🤦‍♂️